ipld/hamt/src/node.rs

```diff
@@ -257,7 +259,14 @@ where
 
         // No existing values at this point.
         if !self.bitfield.test_bit(idx) {
-            self.insert_child(idx, key, value);
+            if conf.min_data_depth <= depth {
```

Comment:
Nit: this would be more obviously correct to me if the test were written the other way round

Reply:
Interesting, thanks for sharing your preference. I have that same feeling that I want the variable to be on the left, but I also like to order GT and LT relations as if they were aligned on an axis, like `x < y && y < z`. 

I can reverse it, no problem.